### PR TITLE
pw-cat, pw-play, pw-record: add page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by Managor** [comment]
Added the fedora wiki as a placeholder until we get better official documentation.

Does anyone have opinions on adding `pw-cat -p --volume=0.1 <audiofile.wav>` as redundancy?

**Comment by Managor** [comment]
There's also `pw-midiplay` and `pw-midirecord`, but I've yet to figure how to make them useful.

File: pages/linux/pw-cat.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+> Pipewire tool for playing and recording audio files.
+> More information: <https://fedoraproject.org/wiki/QA:Testcase_PipeWire_PipeWire_CLI>.
+
+- List all available playback targets:
+
+`pw-cat --playback --list-targets`
+
+- Play a wav sound file over the default target:
+
+`pw-cat --playback {{audiofile.wav}}`
```
Comment (navarroaxel) [inline pages/linux/pw-cat.md:12]: We use `path/to/file` for placeholders:
```suggestion
`pw-cat --playback {{path/to/file.wav}}`
```

File: pages/linux/pw-cat.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- List all available record targets:
+
+`pw-cat --record --list-targets`
+
+- Record a sample recording at a different volume level:
+
+`pw-cat --record --volume={{0.1}} {{quiet.wav}}`
```
Comment (navarroaxel) [inline pages/linux/pw-cat.md:20]:
```suggestion
`pw-cat --record --volume={{0.1}} {{path/to/file.wav}}`
```

File: pages/linux/pw-play.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+> Shorthand tool for pw-cat --playback.
+> More information: <https://fedoraproject.org/wiki/QA:Testcase_PipeWire_PipeWire_CLI>.
+
+- List all available playback targets:
+
+`pw-play --list-targets`
+
+- Play a wav sound file over the default target:
+
+`pw-play {{audiofile.wav}}`
```
Comment (navarroaxel) [inline pages/linux/pw-play.md:12]:
```suggestion
`pw-play {{path/to/file.wav}}`
```

File: pages/linux/pw-play.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+- Play a wav sound file at a different volume level:
+
+`pw-play --volume={{0.1}} {{audiofile.wav}}`
```
Comment (navarroaxel) [inline pages/linux/pw-play.md:16]:
```suggestion
`pw-play --volume={{0.1}} {{path/to/file.wav}}`
```

File: pages/linux/pw-record.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+> Shorthand tool for pw-cat --playback.
+> More information: <https://fedoraproject.org/wiki/QA:Testcase_PipeWire_PipeWire_CLI>.
+
+- List all available record targets:
+
+`pw-record --list-targets`
+
+- Record a sample recording using the default target:
+
+`pw-record {{audiotest.wav}}`
```
Comment (navarroaxel) [inline pages/linux/pw-record.md:12]:
```suggestion
`pw-record {{path/to/file.wav}}`
```

File: pages/linux/pw-record.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+- Record a sample recording at a different volume level:
+
+`pw-record --volume={{0.1}} {{quiet.wav}}`
```
Comment (navarroaxel) [inline pages/linux/pw-record.md:16]:
```suggestion
`pw-record --volume={{0.1}} {{path/to/file.wav}}`
```

File: pages/linux/pw-record.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+- Record a sample recording using a different sample rate:
+
+`pw-record --rate={{6000}} {{lowrate.wav}}`
```
Comment (navarroaxel) [inline pages/linux/pw-record.md:20]:
```suggestion
`pw-record --rate={{6000}} {{path/to/file.wav}}`
```

File: pages/linux/pw-cat.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+`pw-cat --record --list-targets`
+
+- Record a sample recording at a different volume level:
+
+`pw-cat --record --volume={{0.1}} {{path/to/file.wav}}`
+
+- Record a sample recording using a different sample rate:
+
+`pw-cat --record --rate={{6000}} {{lowrate.wav}}`
```
Comment (navarroaxel) [inline pages/linux/pw-cat.md:24]:
```suggestion
`pw-cat --record --rate={{6000}} {{path/to/file.wav}}`
```

File: pages/linux/pw-cat.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
+# pw-cat
+
+> Pipewire tool for playing and recording audio files.
+> More information: <https://fedoraproject.org/wiki/QA:Testcase_PipeWire_PipeWire_CLI>.
+
+- List all available playback targets:
+
+`pw-cat --playback --list-targets`
+
+- Play a wav sound file over the default target:
```
Comment (nicokosi) [inline pages/linux/pw-cat.md:10]: What about?
```suggestion
- Play a WAV file over the default target:
```
https://en.wikipedia.org/wiki/WAV

**Review by nicokosi — APPROVED** [review]
👍

**Review by blueskyson — APPROVED** [review]
Thanks for contribution.

**Pull request merged**